libraries/AP_NMEA_Output/AP_NMEA_Output_config.h

```diff
@@ -0,0 +1,22 @@
+#include <AP_HAL/AP_HAL_Boards.h>
+
+// Needs SerialManager + (AHRS or GPS)
+
+#ifdef HAL_BUILD_AP_PERIPH
+    #define AP_NMEA_OUTPUT_LIBRARY_REQUIREMENTS (defined(HAL_GCS_ENABLED) && HAL_GCS_ENABLED && defined(HAL_PERIPH_ENABLE_GPS))
```

Comment:
`HAL_GCS_ENABLED` should always be defined?

Reply:
fixed